system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -34,6 +34,17 @@
                   :ContactPersons [{:Roles ["Data Center Contact"]
                                     :LastName "Smith"}]}]})
 
+(def update-replace-keywords-umm
```

Comment:
Since we are separating the update and the replace, maybe it's better to name it more clearly?